R/lt_id.R

```diff
@@ -291,7 +291,7 @@ lt_id_Ll_S      <- function(nLx, lx = NULL, Age, AgeInt = NULL, N = 5) {
     radix <- ifelse(nLx[1]>1, 10^nchar(trunc(nLx[1])), 1)  
   }
   # validate nLx
-  stopifnot(all(nLx>0, nLx[-n] < (radix*N)))
+  stopifnot(all(nLx>=0, nLx[-n] < (radix*N)))
```

Comment:
hmmm. could we perhaps do `>=` for all ages (new) plus `>` for ages above some threshold?

Reply:
Something like nLx>=0 & nLx[Age<100]>0 ?
Checking first same length on nLx and Age (probably already checked in parent function)